tests/test_core.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+import nomenclature as nc
+
+
+def test_nonexisting_path_raises():
```

Comment:
If you change the test in `nomenclature/core.py` to check for a directory, this needs of course to be reflected in the unittest

Reply:
done